secprofile/oci-profile.go

```diff
@@ -118,7 +119,7 @@ func allowSyscallWithArgs(seccompProfile *specs.LinuxSeccomp, syscallName types.
 				if syscallName == syscallNameToAllowStr &&
 					((len(syscallArgs) == 0 && len(syscallRule.Args) == 0) ||
 						reflect.DeepEqual(syscallRule.Args, syscallArgs)) {
-					logrus.Errorf("Syscall already added: %s", syscallNameToAllowStr)
+					logrus.Debugf("Syscall already added to Seccomp profile: %s", syscallNameToAllowStr)
```

Comment:
Is it possible to add a test for this case when the syscall is already added to the profile?

Reply:
will fix.